### PR TITLE
Pull request for devel

### PR DESCRIPTION
**Pull request by pabelanger**
Rename EE_BASE_IMAGE to ANSIBLE_RUNNER_IMAGE
Rename EE_BUILDER_IMAGE to ANSIBLE_BUILDER_IMAGE

### CODE REVIEW

File: ansible_builder/constants.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 build_arg_defaults = dict(
     ANSIBLE_GALAXY_CLI_COLLECTION_OPTS='',
-    EE_BASE_IMAGE='quay.io/ansible/ansible-runner:devel',
-    EE_BUILDER_IMAGE='quay.io/ansible/ansible-builder:latest'
+    ANSIBLE_RUNNER_IMAGE='quay.io/ansible/ansible-runner:devel',
```
Comment (AlanCoding) [inline ansible_builder/constants.py:17]: Raising this point out of a sense of obligation:

https://github.com/ansible/ansible-builder/issues/213

If we, in the future, allow building EEs with other EEs as the base image (as opposed to the ansible-runner image), then this naming will age poorly.

How interested are we in that feature? I'm not sure. If we don't intend to support it, then your naming here would help keep people from trying it.

**Review by AlanCoding — APPROVED** [review]
If you merge this, would you be willing to close #213? If yes, then merge.

**Comment by pabelanger** [comment]
@AlanCoding I'm not sure, the plan for downstream was to start to build things atop of an EE, to reduce the duplication of dependency with multiple builds over layered approach.

Given I didn't raise it, will have to differ to @anshulbehl

**Comment by pabelanger** [comment]
that said, we really need to understand if we want to change this to redhat product specific naming or more upstream specific images.  Either choice is going to cause confusion for people.

**Pull request closed without merging**